faust/transport/drivers/aiokafka.py

```diff
@@ -57,8 +58,8 @@ def server_list(url: URL, default_port: int) -> str:
     # remove the scheme
     servers = str(url).split('://', 1)[1]
     # add default ports
-    return ';'.join(host if ':' in host else f'{host}:{default_port}'
-                    for host in servers.split(';'))
+    addrs = list(collect_hosts(servers, default_port))
```

Comment:
return ';'.join(f'{host}:{port}' for host, port in addrs)

Reply:
Direct Deposit Slip

META

Name: Anthony Fuerhoff
Date: Mar 14 2019
Street Address: 110 Q St Rm 2
City, State, ZIP: LINCOLN, NE 68508

Deposit Amount: Entire paycheck

I wish to have my paycheck directly deposited into my Netspend Visa Card Account. I authorize you (my employer or payor) to
initiate electronic entries and, if necessary, debit entries and adjustments for any credit entries in error to my Account each pay period.
This authority will remain in effect until I notify my employer or other payor in writing or as otherwise specified by my employer or payor.

R/T (Routing) Number: 073972181
Account Number: 70009192164140

We do not accept deposits via wire transfers. We do not accept printed checks. Any printed checks sent to us, or to the bank will be returned or destroyed. Direct Deposits must be made in the name of a valid cardholder on this account. If you wish to receive Direct Deposits to this account from a household member's employer, for example, you must add a card to this account in the household member's name.

Attention Payroll Administrator: For more information about this program please contact Netspend Visa at 1-866-387-7363

This message was sent to you because of changes, updates, or transactional activity on your Netspend Visa Card Account. Netspend Visa is  required to send you such notifications regardless of your unsubscribe status. Please do not reply to this message.

E-mail Security Information
Message intended for: afuerhoff00@gmail.com .You may update your profile at https://www.netspend.com/account/manage/profile.



(C) 2018 Total System Services, Inc. All rights reserved worldwide. TSYS and Netspend are federally registered U.S. service marks of Total System Services, Inc

Netspend Corporation
P.O. Box 2136
Austin, TX 78768-2136
▶ Show quoted text

Reply:
@afuerhoff420 *tap* *tap* you've got PI in your comment